src/Types.h

```diff
@@ -13,7 +13,7 @@
 #define CPU_DATA_REGISTRY_LIST_SIZE (8)
 
 ////////////////////////////
-///  FR = Flag Register  ///
+///  FR = Flag Register2  ///
```

Comment:
Global Rename is dangerous

```suggestion
///  FR = Flag Register  ///
```